app/assets/javascripts/discourse/app/components/relative-date.gjs

```diff
@@ -0,0 +1,32 @@
+import Component from "@glimmer/component";
+import { longDate, relativeAge } from "discourse/lib/formatter";
+
+export default class RelativeDate extends Component {
+  get datetime() {
+    if (this.memoizedDatetime) {
+      return this.memoizedDatetime;
+    }
+
+    this.memoizedDatetime = new Date(this.args.data.date);
```

Comment:
It sucks we have the have `this.args.data.date` here instead of `this.args.date`, but the issue is we can't splat args.... so to use this for it's first use case, we'd have to have _another_ component just to wrap this component to avoid the `data`.. idk.